pkg/views/workspace/selection/repository.go

```diff
@@ -14,25 +15,20 @@ import (
 	tea "github.com/charmbracelet/bubbletea"
 )
 
-func selectRepositoryPrompt(repositories []apiclient.GitRepository, index int, choiceChan chan<- string, parentId string, selectedReposParentMap map[string]bool, selectedRepos map[string]bool) {
+func selectRepositoryPrompt(repositories []apiclient.GitRepository, index int, choiceChan chan<- string, selectedRepos map[string]int) {
 	items := []list.Item{}
-	disabledReposCount := 0
 
 	// Populate items with titles and descriptions from workspaces.
 	for _, repository := range repositories {
 		url := *repository.Url
-		title := *repository.Name
-		isDisabled := false
 
 		// Index > 1 indicates use of 'multi-project' command
-		if index > 1 && len(selectedRepos) > 0 && selectedRepos[url] {
-			title += statusMessageDangerStyle(" (Already selected)")
-			// isDisabled property helps in skipping over this specific repo option, refer to
-			// handling of up/down key press under update method in ./view.go file
-			isDisabled = true
-			disabledReposCount++
+		// Append occurence number so as to keep unique names for duplicate entries.
+		if index > 1 && len(selectedRepos) > 0 && selectedRepos[url] > 0 {
+			*repository.Name += strconv.Itoa(selectedRepos[url] + 1)
```

Comment:
![Screenshot 2024-07-26 at 09 38 47](https://github.com/user-attachments/assets/1a9a257a-eb8f-4bce-aa9e-6be5e9d151c1)

This is not intended. You are modifying the name of the repository which is confusing for users.

My previous comment was hinting to modifying the **project** name once the repo is selected, not the repository.

Reply:
do you mean, after all repo selections change the name and changed names to be seen for the f10 view ?

Reply:
![Screenshot 2024-07-26 at 12 21 44](https://github.com/user-attachments/assets/590e5ed0-2a9b-4214-b840-0f2d992451a2)

Yes, when the user gets to this view, the project list should already have the updated names so they are correct in the f10 view.

Reply:
done, Up for review!